pydash/pydash_app/dashboard/endpoint.py

```diff
@@ -20,9 +21,11 @@ def __init__(self, name, is_monitored):
 
         self.id = uuid.uuid4()
         self.name = name
-        self.calls = list()
         self.is_monitored = is_monitored
 
+        self._calls = []
+        self._aggregator = Aggregator(self._calls)
```

Comment:
There is another problem with this line: The aggregator will be initialized with a reference to exactly the same list as `endpoint._call`. 

Observe:
```python
> a = [1,2,3]
> b = a
> a.append(4)
> b
[1,2,3,4]
```

In this case this means that a `call` is added twice when line 40+41 are executed.

(That is this part:
```python
        self._calls.append(call)
        self._aggregator.add_endpoint_call(call)
```